docs/topics/db/aggregation.txt

```diff
@@ -323,6 +323,16 @@ title that starts with "Django" using the query::
 
     >>> Book.objects.filter(name__startswith="Django").aggregate(Avg('price'))
 
+When filtering against :ref:`multiple values <spanning-multi-valued-relationships>`,
+be careful to structure your filters according to the behavior you desire::
+
+    # Annotated list of the intersection of all books with *both* a title starting with "Django" and with more than 100 pages.
+    >>> Book.objects.filter(name__startswith="Django",
+                    pages__gt=100).annotate(num_authors=Count('authors'))
+    # Annotated list of the union of all books with a title starting with "Django" and all books with more than 100 pages.
+    >>>Book.objects.filter(name__startswith="Django").filter(
+                    pages__gt=100).annotate(num_authors=Count('authors'))
+
```

Comment:
I don't think they will be, at least not according to the documentation [here] (https://docs.djangoproject.com/en/1.9/topics/db/queries/#spanning-multi-valued-relationships) and the original [ticket] (https://code.djangoproject.com/ticket/19415), unless I've misunderstood something.